config/the-config_top-section.md

```diff
@@ -111,10 +111,13 @@ For Curved hilts, or where the board is not parallel with the blade (for Twist o
 This will rotate the orientation of the board 20 degrees around the Y axis.
 Depending on the orientation of the board, you might need -20 degrees instead. (Or 15 degrees? or 30?)
 
+
 ```cpp
 #define ORIENTATION_ROTATION 0,20,0
 ```
 
+The values for the define are in degrees and in the order of `X,Y,Z`, to see how this translates to physical rotation of the board, see [board_orientation_rotation.svg](https://pod.hubbe.net/images/board_orientation_rotation.svg).
```

Comment:
1. the image should be inline, not a link. (Or a separate page, with an inline image)
2. Are you sure the rotational directions are correct? (did you test it?)
3. It would be fantastic to have example pictures as well. Something like: "If your board is angled this way (picture) then ORIENTATION_ROTATION should be 0,-20,0"

Reply:
> the image should be inline, not a link. (Or a separate page, with an inline image)
Oh, ok... I was going off how the orientation define image was done. How would you want it inlined? via HTML to set the width or just with normal markdown? (How big do you want it?)

> Are you sure the rotational directions are correct? (did you test it?)
This was an oversight, I did not, testing now seems to show that X and Y are actually switch from those descriptions. Will also update for rotation direction, that thought didn't even occur to me.

> It would be fantastic to have example pictures as well
Hmm... that might be difficult, I'll see what I can do.

Reply:
Hmm, we should probably fix the orientation defines image too, but that's a separate issue.

https://pod.hubbe.net/making_changes.html#images

Reply:
> Hmm, we should probably fix the orientation defines image too, but that's a separate issue.
> 
> https://pod.hubbe.net/making_changes.html#images

Done.